### PR TITLE
Add recap post for zero-cost reporting

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _posts/2023-06-28-trino-fest-2023-starburst-recap.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Zero-cost reporting"
+author: "Jan Waś, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Starburst.png
+show_pagenav: false
+---
+
+Let's say you have some data. Maybe it's in a spreadsheet, a CSV file, a
```
Comment (mosabua) [inline _posts/2023-06-28-trino-fest-2023-starburst-recap.md:10]: I feel like this paragraph is a bit long to be all before the "more" break. Can we separate it up somehow?

File: _posts/2023-06-28-trino-fest-2023-starburst-recap.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+relational database, or multiple terabytes of data in an S3 bucket. You need
+to run SQL queries on this data, and you'd like to share those results with your
+teammates, coworkers, and partner teams, but you want to do it in a way that
+allows everyone to view those results on-demand, on the web, and with the latest
+results without the need for any manual effort on your part. There are a lot of
```
Comment (mosabua) [inline _posts/2023-06-28-trino-fest-2023-starburst-recap.md:15]: Maybe break here?

File: _posts/2023-06-28-trino-fest-2023-starburst-recap.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+need to spend time or money to set it up, and you don't want to spend a lot.
+With so many options, there's the possibility to get stuck in analysis
+paralysis, and trying to find the best way forward may leave you stuck forever.
+Jan Waś from Starburst has a suggestion: keep it simple with Trino, plaintext
+files, Git, and GitHub actions, and you can set it all up for free.
+
+<!--more-->
+
+{% youtube 586qvEyuO_U %}
+
```
Comment (mosabua) [inline _posts/2023-06-28-trino-fest-2023-starburst-recap.md:26]: Slide deck is missing

**Review by mosabua — CHANGES_REQUESTED** [review]
Two things only .. pretty much ready to roll.

File: _posts/2023-06-28-trino-fest-2023-starburst-recap.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+Let's say you have some data. Maybe it's in a spreadsheet, a CSV file, a
+relational database, or multiple terabytes of data in an S3 bucket. You need
+to run SQL queries on this data, and you'd like to share those results with your
+teammates, coworkers, and partner teams, but you want to do it in a way that
+allows everyone to view those results on-demand, on the web, and with the latest
+results without the need for any manual effort on your part. There are a lot of
+tools that might be able to do this for you, but whatever you choose, you'll
+need to spend time or money to set it up, and you don't want to spend a lot.
```
Comment (ryandoowan) [inline _posts/2023-06-28-trino-fest-2023-starburst-recap.md:17]: The second part feels a bit redundant

File: _posts/2023-06-28-trino-fest-2023-starburst-recap.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+With so many options, there's the possibility to get stuck in analysis
```
Comment (ryandoowan) [inline _posts/2023-06-28-trino-fest-2023-starburst-recap.md:18]: possibility of getting stuck

**Review by mosabua — APPROVED** [review]

**Review by ryandoowan — APPROVED** [review]

**Pull request merged**